src/containers/User/UserHeader.module.scss

```diff
@@ -12,6 +12,35 @@
   flex-shrink: 0;
 }
 
+.a{
+  text-decoration: none;
+  color: lightgray;
+}
+
+.link a{
+  font-weight: normal;
+  text-decoration: none;
+  color: lightgray;
+  transition: color .15s ease-in;
```

Comment:
I think there should be a hover color on the link 
var(--color-secondary)
It keeps the color code more logical throughout the app